cdc/processor/pipeline/mounter.go

```diff
@@ -14,42 +14,119 @@
 package pipeline
 
 import (
-	"github.com/pingcap/ticdc/cdc/entry"
+	"context"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/edwingeng/deque"
+	"github.com/pingcap/failpoint"
+	"github.com/pingcap/ticdc/cdc/model"
+	"github.com/pingcap/ticdc/pkg/notify"
 	"github.com/pingcap/ticdc/pkg/pipeline"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+	"golang.org/x/time/rate"
+)
+
+const (
+	waitEventMountedBatchSize = 128 * 1024
+	maxNotificationsPerSecond = 2.0
```

Comment:
The message batch size, notification rate and check interval (100ms), how can these parameters affect the cpu usage and processing throughput